### PR TITLE
EXP: Cleanup Code

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by Volco** [comment]
@DeathlyPlays is this all tested and you're 100% it works no crashes?

**Comment by DeathlyPlays** [comment]
Yes @Volco

File: wavelength-plugins/exp.js [hunk 1]
```diff
@@ -220,44 +227,47 @@ exports.commands = {
...
 		if (Users.get(target)) Users.get(target).popup('Your XP was reset by an Administrator. This cannot be undone and nobody below the rank of Administrator can assist you or answer questions about this.');
-		user.popup("|html|You have reset the XP of " + WL.nameColor(targetUser, true) + ".");
-		Monitor.adminlog('[EXP Monitor] ' + user.name + ' has reset the XP of ' + target);
+		user.popup(`|html|You have reset the XP of ${WL.nameColor(targetUser, true)}.`);
+		Monitor.adminlog(`[EXP Monitor] ${user.name} has reset the XP of ${target}`);
 		room.update();
 	},
 
-	doublexp: 'doubleexp',
+	doublexp: "doubleexp",
```
Comment (Volco) [inline wavelength-plugins/exp.js]: why is this ``"`` and the others are ``'``?

File: wavelength-plugins/exp.js [hunk 2]
```diff
@@ -220,44 +227,47 @@ exports.commands = {
...
 	doubleexp: function (target, room, user) {
-		if (!this.can('exp')) return;
+		if (!this.can("exp")) return;
```
Comment (Volco) [inline wavelength-plugins/exp.js]: same as other comment

File: wavelength-plugins/exp.js (hunk 1 above)
Comment (DeathlyPlays) [reply wavelength-plugins/exp.js]: Looks cleaner imo

Comment (Volco) [reply wavelength-plugins/exp.js]: my question is... why are the other alias' using ``'`` when this one is the only one using ``"``

**Comment by Volco** [comment]
@DeathlyPlays fully tested? fully works?

**Comment by DeathlyPlays** [comment]
yup

**Pull request merged**